### DISCUSSION TITLE
Layout dot delimiter does not seem to work when $ is used in the filename

### DISCUSSION THREAD

**Discussion by dan-cooke** [remix-run/remix#2397] [Q&A] [answered]
I have the following routes

```
portfolios/
    - $id.tsx -- layout for viewing a portfolio
    -  $id/
            - holdings.tsx -- page nested inside a portfolio
    - created.tsx
portfolios.tsx -- layout for portfolios
```

I would like to use the [dot delimiter](https://remix.run/docs/en/v1/api/conventions#dot-delimeters) to break up this layout, as I have the following requirement:

- `portfolios.tsx` layout should not be used in the nested `$id.tsx` layout

So using the docs I converted my routes as follows:

```
portfolios/
    - created
portfolios.$id.holdings
portfolios.$id - i would like this to be the layout route for `portfolios/$id
portfolios.tsx - and this to be the layout route for the portfolios directory
```

But this results in my `portfolios.$id.holdings` not rendering its layout route `portfolios.$id`

**Comment by machour** [comment, marked as answer]
I think you want this instead:

```
portfolios.tsx // layout
portfolios/
  - index.tsx   <== /portfolios/
  - created.tsx <== /portfolios/created
portfolios.$id.tsx // layout
portfolios.$id/ 
  - index.tsx <== /portfolios/4
  - holdings.tsx  <== /portfolios/4/holdings
```

**Reply by dan-cooke** [reply]
EDIT: never mind, i missed the key piece in your example and its working now! Thank you
